engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/ProcessDefinition.xml

```diff
@@ -295,6 +301,12 @@
       <if test="deploymentId != null">
         and RES.DEPLOYMENT_ID_ = #{deploymentId}
       </if>
+      <if test="deployedAfter != null">
+        and DEP.DEPLOY_TIME_ > #{deployedAfter}
```

Comment:
The `>` probably needs to be URL encoded to be valid XML?! We use `&gt;` in other places.